src/ocamlorg_frontend/package_intf.ml

```diff
@@ -1,16 +1,29 @@
+type version = Latest | Version of string
```

Comment:
Are you using this? I can't find where. If you do, consider renaming `Version` into something like `Numbered`, `Released` or `Named` to avoid using the same word twice.